_posts/2022-11-21-vws-rse-cop-summary.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Encouraging and Enabling Mutual Ownership in a RSE Community of Practice
+tags: [2022 virtual workshop]
+posted_by: Miranda Mundt
+---
+
+
+_Authors: Miranda Mundt, Jonathan Bisila, Jon Guyer, Daniel Howard,
+Daniel S. Katz, Reed Milewicz, Henry Schreiner, Joshua Teves, Chris Wiswell_
+
+## Introduction
+
+The explosion of Research Software Engineers (RSEs) in the United States
+created the opportunity to form communities of practice (CoP), groups which
+share a passion for an activity and learn how to do it better as they interact
+regularly, specifically to benefit their local organization or region. CoPs,
+however, only succeed insofar as their members feel heard, included, and valued.
+
+As part of the [2022 US-RSE Virtual Workshop](https://us-rse.org/virtual-workshop-2022/),
+Jonathan Bisila, Reed Milewicz, and Miranda Mundt of Sandia National Laboratories
+moderated the breakout session “Encouraging and Enabling Mutual Ownership in a
+RSE Community of Practice.” In this session, participants from industry,
+academia, and national laboratories and agencies across the country joined
+together to create actionable suggestions for how to get members involved in a
+community of practice and how to make them feel invested and included. They
+hope that the results of this discussion can assist other RSEs in the
+creation of their own local CoP.
+
+## Starting a community of practice
+
+A CoP is a group of people who share some concern or passion for what they do
+and want to learn to do it better as they interact regularly. CoPs generally
+have 3 elements: (1) mutual engagement, (2) a joint enterprise, and (3) a
+shared repertoire. The three moderators recently took on the challenge of
+creating a formally-defined CoP at Sandia National Laboratories.
```

Comment:
is the CoP at Sandia related to RSEng?  If so, maybe say so here. Otherwise, the question in the next paragraph is a bit confusing.

Reply:
I'm inclined to disagree with you here. The introduction clearly explains that this post is about making an RSE CoP.

Reply:
ok - if no one else sees this the way I do, that's fine